### PR TITLE
Add v5.1 mpp tpch test report

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
Add v5.1 mpp tpch test report

The alias for the deleted `/zh/tidb/dev/v5.0-performance-benchmarking-with-tpch` file is added in https://github.com/pingcap/docs-cn/pull/6511.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [x] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: benchmark/v5.1-performance-benchmarking-with-tpch.md [hunk 1]
```diff
@@ -112,4 +112,4 @@ Apache Spark 测试使用 Apache Parquet 作为存储格式，数据存储在 HD
 
 ![TPC-H](/media/tidb-v5-tpch-100-vs-gp-spark.png)
```
Comment (qiancai) [inline benchmark/v5.1-performance-benchmarking-with-tpch.md:113]: @ilovesoup Would you please share the Google sheets with this graphic so we can update v5.0 in the graphic to v5.1?

File: benchmark/v5.1-performance-benchmarking-with-tpch.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 
-# TiDB TPC-H 性能对比测试报告 - v5.0 MPP 模式对比 Greenplum 6.15.0 以及 Apache Spark 3.1.1
+# TiDB TPC-H 性能对比测试报告 - v5.1 MPP 模式对比 Greenplum 6.15.0 以及 Apache Spark 3.1.1
 
 ## 测试概况
 
-本次测试对比了 TiDB v5.0 MPP 模式下和主流分析引擎例如 Greenplum 和 Apache Spark 最新版在 TPC-H 100 下的性能表现。结果显示，TiDB v5.0 MPP 模式下相对这些方案有 2-3 倍的性能提升。
+本次测试对比了 TiDB v5.1 MPP 模式下和主流分析引擎例如 Greenplum 和 Apache Spark 最新版在 TPC-H 100 下的性能表现。结果显示，TiDB v5.1 MPP 模式下相对这些方案有 2-3 倍的性能提升。
 
 TiDB v5.0 中新引入了 [TiFlash](/tiflash/tiflash-overview.md) 组件的 MPP 模式，这大幅增强了 TiDB HTAP 形态。本文的测试对象如下：
```
Comment (qiancai) [inline benchmark/v5.1-performance-benchmarking-with-tpch.md:11]: Do we need to update this sentence for v5.1 to show the stability improvement of 5.1 MPP? @ilovesoup

Comment (ilovesoup) [reply benchmark/v5.1-performance-benchmarking-with-tpch.md:11]: There is no update in TPC-H directly in MPP in 5.1. I think update like this is ok.

File: benchmark/v5.1-performance-benchmarking-with-tpch.md (hunk 1 above)
Comment (TomShawn) [reply benchmark/v5.1-performance-benchmarking-with-tpch.md:113]: @qiancai A simple image processing can also do the trick. PTAL https://github.com/pingcap/docs-cn/blob/eeda5575e0a710443456c1ea6d4b2f4bdd352d27/media/tidb-v5-tpch-100-vs-gp-spark.png

**Review by qiancai — APPROVED** [review]

File: TOC.md [hunk 3]
```diff
@@ -13,7 +13,7 @@
   + 性能测试报告
     + [Sysbench 性能对比 - v5.1 对比 v5.0](/benchmark/benchmark-sysbench-v5.1.0-vs-v5.0.2.md)
     + [TPC-C 性能对比 - v5.1 对比 v5.0](/benchmark/v5.1-performance-benchmarking-with-tpcc.md)
-    + [TPC-H 100 性能对比 - v5.0 MPP 对比 Greenplum / Apache Spark](/benchmark/v5.0-performance-benchmarking-with-tpch.md)
+    + [TPC-H 100 性能对比 - v5.1 MPP 对比 Greenplum / Apache Spark](/benchmark/v5.1-performance-benchmarking-with-tpch.md)>>>>>>> master
```
Comment (TomShawn) [inline TOC.md:16]:
```suggestion
    + [TPC-H 100 性能对比 - v5.1 MPP 对比 Greenplum / Apache Spark](/benchmark/v5.1-performance-benchmarking-with-tpch.md)
```

**Review by ilovesoup — APPROVED** [review]
LGTM

**Pull request merged**